lib/config/apiConfig.js

```diff
@@ -17,4 +17,5 @@ module.exports = {
     pagination: {
         limit: pageItemsLimit,
     },
+    searchItemsSeparator: ',',
```

Comment:
As I said, this should not be present (or at least not in this form). Please leave this in legacy code.

Reply:
revoked